misc/pytorch_toolkit/text_recognition/vocabs/vocab_alphanumeric.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
do you really need these vocabs? Having just one alphabet string is not enough?

Reply:
For alphanumeric it will be enough, but not for the formula recognition, where atomic units could consist of several letters (like `\sin`)